sdk-ext-ui-lib/src/androidTest/java/cash/z/ecc/sdk/ext/ui/model/FiatCurrencyConversionRateStateTest.kt

```diff
@@ -14,6 +16,7 @@ import kotlin.time.Duration.Companion.seconds
 class FiatCurrencyConversionRateStateTest {
     @Test
     @SmallTest
+    @SdkSuppress(minSdkVersion = Build.VERSION_CODES.O)
```

Comment:
@ccjernigan this is a temporary solution to avoid failing our tests on Android SDK 25 and lower, as kotlinx.datetime.Instant class supports SDK 26 and higher (our lib modules are now on 24), but I think we should introduce different solution, e.g. change our libs SDK version to 26 (App module is already on 27), or have fallback solution for lower SDKs. What do you think?

Reply:
There's a simple solution.  See app.build.gradle.kts:

    compileOptions {
        isCoreLibraryDesugaringEnabled = true
    }

and

    coreLibraryDesugaring(libs.desugaring)

Turning that on for the library module should solve it.